extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeAbbreviationSubCountMap.java

```diff
@@ -0,0 +1,136 @@
+package org.schabi.newpipe.extractor.services.youtube.extractors;
+
+import java.util.HashMap;
+
+public class YoutubeAbbreviationSubCountMap {
+
+    //should be safe until someone has 1 billion subscribers on YouTube
+    public static final HashMap<String, String> abbreviationSubscribersCount = new HashMap<>();
+    public static String englishMillionAbbreviation = "M";
+    public static String englishThousandAbbreviation = "K";
+    public static String tenThousandAbbreviation = "万";
+    public static String hundredThousandAbbreviation = "ল";
+    public static String tenMillionAbbreviation = "ক";
+    public static String hundredMillionAbbreviation = "億";
+
+    static {
+        abbreviationSubscribersCount.put(englishThousandAbbreviation, englishThousandAbbreviation); //az, iw, en
+        abbreviationSubscribersCount.put(englishMillionAbbreviation, englishMillionAbbreviation); //iw, en
+        abbreviationSubscribersCount.put(tenThousandAbbreviation, tenThousandAbbreviation);
+        abbreviationSubscribersCount.put(hundredMillionAbbreviation, hundredMillionAbbreviation);
+        abbreviationSubscribersCount.put(tenMillionAbbreviation, tenMillionAbbreviation);
+        abbreviationSubscribersCount.put(hundredMillionAbbreviation, hundredMillionAbbreviation);
+
+        abbreviationSubscribersCount.put(" k", englishThousandAbbreviation); //af
+        abbreviationSubscribersCount.put(" m", englishMillionAbbreviation); //af, is
+        abbreviationSubscribersCount.put(" ሺ", englishThousandAbbreviation); //am
+        abbreviationSubscribersCount.put(" ሜትር", englishMillionAbbreviation); //am
+        abbreviationSubscribersCount.put(" ألف", englishThousandAbbreviation); //ar
+        abbreviationSubscribersCount.put(" مليون", englishMillionAbbreviation); //ar
+        abbreviationSubscribersCount.put(" mln", englishMillionAbbreviation); //az, et, lt, nl, pl, sq, uz
+        abbreviationSubscribersCount.put(" тыс", englishThousandAbbreviation); //be, ru
+        abbreviationSubscribersCount.put(" млн", englishMillionAbbreviation); //be, bg, kk, ky, ru, uk
+        abbreviationSubscribersCount.put(" хил", englishThousandAbbreviation); //bg
+        abbreviationSubscribersCount.put(" হা", englishThousandAbbreviation); //bn
+        abbreviationSubscribersCount.put(" hilj", englishThousandAbbreviation); //bs, sr
+        abbreviationSubscribersCount.put(" mil", englishMillionAbbreviation); //bs, cs, hr, ro, sk, sr-Latn
+        abbreviationSubscribersCount.put("m", englishThousandAbbreviation); //ca
+        abbreviationSubscribersCount.put(" M", englishMillionAbbreviation); //ca, es, eu, and many more
```

Comment:
Why have some keys a leading whitespace and some not?

Reply:
Because abbreviations are the same as Youtube give, except the final dot that is removed by the function removeNumber
Eg,
English 100K, French 100 K, German 100 Mio., so in the map there is also the whitespace.
And I guess and hope YouTube respects the punctuation, and punctuation changes with languages (for example in French I would put space before !, but in English it's stickied)